packages/config/src/Config.js

```diff
@@ -25,8 +25,8 @@ export class Config {
     }
   }
 
-  _merge(newValue, onError = false) {
-    const curVales = deepMerge.all([this.#config, newValue]);
+  _merge({ defaultValue = null, newValue, onError = false }) {
+    const curVales = deepMerge.all([defaultValue || this.#config, newValue]);
```

Comment:
im not sure what's going on here.
also a test on that matter would be better to help us understand what's happening and when